client/src/app/pages/controls/stakeholder-groups/components/stakeholder-group-form.tsx

```diff
@@ -109,6 +109,55 @@ export const StakeholderGroupForm: React.FC<StakeholderGroupFormProps> = ({
     mode: "onChange",
   });
 
+  const onCreateStakeholderGroupSuccess = (
+    _: AxiosResponse<StakeholderGroup>
+  ) =>
+    pushNotification({
+      title: t("toastr.success.create", {
+        type: t("terms.stakeholderGroup"),
+      }),
+      variant: "success",
+    });
+
+  const onCreateStakeholderGroupError = (error: AxiosError) => {
```

Comment:
Can you move all unrelated work to new PRs with a description & intention behind the changes? Thanks.

Reply:
Same response

Reply:
Most of these changes are out of the scope of this PR. I know how it goes when you get rolling and want to make mass changes. But we should be looking to avoid this type of PR as a team. If we want to convert all of the rest.ts calls to remove the deprecated APIClient, then we should make a PR for that etc.

Reply:
I understand, and I agree with smaller PRs.
Again the cascade of change wasn't triggered to replace APIClient, but I made changes around then a aligned those.